programs/util.h

```diff
@@ -25,12 +25,17 @@ extern "C" {
 #include <stdio.h>        /* fprintf */
 #include <sys/types.h>    /* stat, utime */
 #include <sys/stat.h>     /* stat, chmod */
-#if defined(_MSC_VER)
+#if defined(_WIN32)
```

Comment:
Can you comment on the distinction between `_MSC_VER` and `_WIN32`? Why the change?

I haven't actually used a windows machine in years, so I'm unfamiliar with the dev ecosystem. @Cyan4973?

Reply:
_MSC_VER is only for Visual Studio. MinGW (POSIX wrapper around actual WIndows APIs) does not support utimensat because Windows does not support utimensat.